source/docs/articles/drupal/drupal-distributions-core-version-updates.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Drupal Distributions - Core Version Updates

```

Comment:
I think the title should change to something like "Updating Core on Sites Running Custom Upstreams  that are Slow to Update Core" - Consider also moving it to the `/sites/code` directory, as this will eventually be applicable to WordPress sites in the same situation.